packages/google_identity_services_web/example/id/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+// import 'package:flutter/material.dart';
```

Comment:
Why is there commented out code and empty comments throughout this file?

Reply:
When I copied the main, I commented out a bunch of code, and never looked at that part again. It's gone now!